infrastructure/haproxy_reload.sh

```diff
@@ -1 +1,79 @@
-eval ${HAPROXY_RELOAD_COMMAND}
+#!/bin/bash
+
+num_rules=2
+real=0
```

Comment:
Do you mean script name ? but what it does, it restart haproxy with iptables switch.